### PR TITLE
Add Mend Plugin

### PR DESCRIPTION
**Pull request by NormanWenzelWSS**
Add Mend plugin

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/mend.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Mend.io
+author: dariuszsobkowicz
+authorUrl: https://github.com/dariuszsobkowicz
+category: Security
+description: Proactive application risk management for developer and security teams directly in Backstage
+documentation: https://github.com/backstage/backstage-community-plugins/tree/main/workspaces/mend/plugins/mend
```
Comment (vinzscam) [inline microsite/data/plugins/mend.yaml:7]:
```suggestion
documentation: https://github.com/backstage/community-plugins/tree/main/workspaces/mend/plugins/mend
```

**Review by vinzscam — CHANGES_REQUESTED** [review]
nice! Let's wait until the plugin gets published

**Comment by NormanWenzelWSS** [comment]
> nice! Let's wait until the plugin gets published

@vinzscam @awanlin The plugin was merged into the community plugins repo last Friday. Can you please merge this and publish the plugin package, so that the links in the portal marketplace can be updated (https://backstage.spotify.com/marketplace/mend-io/plugin/mend-io/)?

**Comment by dariuszsobkowicz** [comment]
Hi @awanlin @NormanWenzelWSS, created PR to remove the private flag https://github.com/backstage/community-plugins/pull/1841

**Review by awanlin — APPROVED** [review]
Package has been published on NPM and the CI is green 🚀

**Review by vinzscam — APPROVED** [review]
let's gooo welcome 🤩

**Pull request merged**